### ISSUE TITLE
how to save log file on Windows

### ISSUE THREAD

**Issue by infinitewwj** [AlexeyAB/darknet#298]
for evaluating Avg IOU, Class, Obj, no Obj, Avg Recall, loss after training is done,
I want to save the cmd process when training YOLO on Windows
![image](https://user-images.githubusercontent.com/22359714/33748595-64554352-dc04-11e7-9f30-ac46444a3b3a.png)
but command "darknet.exe detector recall data/voc.data cfg/yolo-voc.2.0.cfg backup/yolo-voc_15000.weights >> yolo-voc-15000.txt " seems not work, the generated txt file is 0KB.
I want to konw if there is another way to save log file?

**Comment by DanLiu0623** [comment]
I use command  on linux:
`./darknet detector recall data/voc.data cfg/yolo-voc.2.0.cfg backup/yolo-voc_15000.weights 2>&1 | tee  yolo-voc-15000.txt`,it works, I do not try it on win~

**Comment by infinitewwj** [comment]
@DanLiu0623 yes, I also used your command on Linux, but I donot know how to save log on Windows

**Comment by AlexeyAB** [comment]
Try to use this: `darknet.exe detector recall data/voc_air.data tiny-yolo-voc.cfg backup/tiny-yolo-voc_5000.weights > z.txt 2>&1`

stdout and stderr will be logged to the z.txt, but will not shown on the display.

**Comment by infinitewwj** [comment]
@AlexeyAB yes, I followed your instruction and it worked. Thanks a lot.
